eng/psakefile.ps1

```diff
@@ -427,3 +430,42 @@ function Create-DocsEnv() {
         deactivate
     }
 }
+
+task check-licenses {
+    # Uses cargo-deny to verify that the linked components
+    # only use approved licenses
+    # https://github.com/EmbarkStudios/cargo-deny
+    Invoke-LoggedCommand -wd $repo.root {
+        cargo deny check licenses
+    }
+}
+
+task update-noticefiles {
+    # use cargo-about to generate a notice files
+    # notice files are only for wheel distributions
+    # as no bundled sources are in the sdist.
+
+    # llvm special license is already in the template
+    # as it is a hidden transitive dependency.
+    # https://github.com/EmbarkStudios/cargo-about
+    $config = Join-Path $repo.root notice.toml
+    $template = Join-Path $repo.root notice.hbs
+    Invoke-LoggedCommand -wd $pyqir.parser.dir {
```

Comment:
Minor, but it feels like these three could be collapsed with a foreach-object over `@($pyqir.parser.dir, $pyqir.generator.dir, $pyqir.evaluator.dir)`?

Reply:
Done